langserver/langserver_test.go

```diff
@@ -157,6 +157,7 @@ func TestServer(t *testing.T) {
 			fs: map[string]string{
 				"a.go":      "package p; var A int",
 				"a_test.go": `package p_test; import "test/pkg"; var X = p.A`,
```

Comment:
Just pushed the commits which do this. We do isolate the two definitions :)

When it comes to actual `UsesOf`, we don't do any normalization on the stuff we get back from loader, so it should be very accurate. The stripping of `_test` we do outside of that is purely heuristic based to help decide which packages deserve having UsesOf run on them.